.gitignore

```diff
@@ -9,4 +9,6 @@ _site
 .Rproj.user
 .Rhistory
 .RData
-
+.vendor/
+.docker-vendor/
```

Comment:
at what step is `.docker-vendor` created?

Reply:
at the `bundle install` step when running `make docker-serve`

Reply:
thank you.